util/encode_modifiers.exs

```diff
@@ -11,11 +12,18 @@ defmodule ModifierEncoder do
     platform_modifiers: @native.platform_modifiers
 
   defmacro quote_argv() do
-    Code.string_to_quoted!(hd(System.argv()))
+    Code.string_to_quoted!(
+      hd(System.argv())
+      |> String.replace(~r'///', "")
+    )
```

Comment:
Mixing nested arguments with pipes diminishes the readability of this code. Consider:

```suggestion
    System.argv()
    |> hd()
    |> String.replace(~r'///', "")
    |> Code.string_to_quoted!()
```